pcsx2/ImGui/FullscreenUI.h

```diff
@@ -7,6 +7,7 @@
 #include "common/ProgressCallback.h"
 #include "common/SmallString.h"
 
+#include <atomic>
```

Comment:
I think this can be moved into the cpp file

Reply:
Done